### PR TITLE
Add aria-live="polite" to error messages so screen readers are prompt…

### PR DESCRIPTION
**Pull request by jtreuting**
…ed to notify the user

Approach based on this: https://cantina.co/form-errors-screen-readers-can-access/

### CODE REVIEW

**Comment by dsnyder-fairway** [comment]
Is this missing from `user-group-form.component.html`and `target-report-form.component.html` as well? This was just seeing what files had `help-block` class in them

**Comment by jtreuting** [comment]
@dsnyder-fairway I'll check.  I was looking for the `.errors` that is used to find the errors, but maybe there are other ways.

File: webapp/src/main/webapp/src/app/report/report-download.component.html [hunk 1]
```diff
@@ -35,6 +35,7 @@ <h4 class="modal-title pull-left">{{title}}</h4>
             </div>
             <div *ngIf="resourceControl.invalid && (resourceControl.dirty || resourceControl.touched)">
               <p *ngIf="resourceControl.errors.required"
+                 aria-live="polite"
                  class="help-block small red">{{'common.form-field-message.required' | translate}}</p>
             </div>
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/report/report-download.component.html]: hm... this is just going to prompt a screen-reader to say "Required" without any indication of which control is required...

Comment (jtreuting) [reply webapp/src/main/webapp/src/app/report/report-download.component.html]: good point.  i initially started it with the error messages that had more context.  i'll do a review of these to see where we might need to add the field label to the required error message.

**Pull request merged**